i18n/es.toml

```diff
@@ -5,7 +5,7 @@ other = "Próximos Eventos"
 other = "Aprende más"
 
 [upcoming-events-learn-more-title]
-other = "Aprender más sobre {{ .Count }}"
+other = "Aprender más sobre {{ . }}"
```

Comment:
upcoming-events-learn-more-title and past-events-learn-more-title are not in use for all three language tomls.

Reply:
But I know this is in use at https://jakartaone.org/